pom.xml

```diff
@@ -9,7 +9,6 @@
         <kotlin.version>1.1.4</kotlin.version>
     </properties>
 
-

```

Comment:
Try to avoid changes like this in the future, just so the changelog stays as concise as possible